Dockerfile

```diff
@@ -17,11 +17,23 @@ COPY controllers/ controllers/
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
 
+# Add labels to image
+LABEL name="Ramen DR operator" \
+  vendor="github.com/RamenDR/ramen" \
+  version="1.0" \
+  summary="Provides disaster recovery and relocations services for workloads and their persistent data" \
+  description="Deploy Ramen DR operator"
+
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+FROM registry.access.redhat.com/ubi8/ubi
```

Comment:
This contradicts the comment - it would be useful to update the comment to explain why
we use ubi instead of the default.

Reply:
Fair point. I'll update this to reflect why this change was made